index.bs

```diff
@@ -367,16 +369,38 @@ QUIC connection.
 
 The [=agent certificate=] must have the following characteristics:
 
-* 256-bit, 384-bit, or 521-bit ECDSA public key
-* Self-signed
-* Supporting the at least one of the following signature algorithms:
-    * secp256r1_sha256
-    * secp384r1_sha384
-    * secp521r1_sha512
-* Valid for signing
+* 256-bit, 384-bit, or 521-bit ECDSA public key.
+* Self-signed.
+* Supporting the at least one of the [=certificate algorithms=] listed below.
```

Comment:
"The at least" seems to be used in legal documents, but I don't think that's a common construct in technical specs.

```suggestion
* Supporting at least one of the [=certificate algorithms=] listed below.
```